src/css/profile/mobile/common/popup.less

```diff
@@ -98,9 +98,22 @@
 			padding-bottom: 24 * @unit_base;
 		}
 		input[type=checkbox] {
-			margin-top: 26 * @unit_base;
-			margin-bottom: -12 * @unit_base;
-			margin-right: 22 * @unit_base;
+			margin-left: 0 * @px_base;
```

Comment:
you can replace this by:
`margin: 0 12 * @px_base 0 0`

Reply:
done.